rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -747,6 +748,38 @@ private void logCommitmentWithVotedFederation(BridgeEventLogger eventLogger, Fed
         logger.debug("[logCommitmentWithVotedFederation] Voted federation committed: {}", votedFederation.getAddress());
     }
 
+    @Override
+    public BtcTransaction createAndSetSvpFundTransactionWithoutSignatures(Coin spendableValueFromProposedFederation, Coin feePerKb, Wallet activeFederationWallet) throws InsufficientMoneyException {
+        Optional<Federation> proposedFederationOpt = provider.getProposedFederation(constants, activations);
+        if (!proposedFederationOpt.isPresent()) {
+            String message = "Proposed federation should be present when creating SVP fund transaction.";
+            logger.warn(message);
+            throw new IllegalStateException(message);
+        }
+        Federation proposedFederation = proposedFederationOpt.get();
+
+        BtcTransaction svpFundTransactionUnsigned = new BtcTransaction(constants.getBtcParams());
+        svpFundTransactionUnsigned.setVersion(BTC_TX_VERSION_2);
+
+        svpFundTransactionUnsigned.addOutput(spendableValueFromProposedFederation, proposedFederation.getAddress());
```

Comment:
There's supposed to be an output for the flyover federation as well, right? Is that part of another task?